Cargo.toml

```diff
@@ -3,3 +3,4 @@ members = [
     "halo2",
     "halo2_proofs",
 ]
+resolver = "2"
```

Comment:
sorry, at some point I bumped the rust version in `rust-toolchain` because rust-analyzer was taking so long and I was hoping a newer version would perform better, and with that I added that line.  I think we should bump the rust version, but that belongs to another PR, so I'll delete this!